### PR TITLE
fix: Dropdown icon to the event creation steps to make it more user friendly.

### PR DESCRIPTION
**Pull request by iamdevvalecha**
Fixes https://github.com/fossasia/open-event-frontend/issues/7384

#### Short description of what this resolves:
fixes the team page by adding dropdown icon & text changed to team members.

#### Changes proposed in this pull request:
Screenshot from 2021-06-04 02-11-19

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamdevvalecha** [comment]
mobile views:-
WhatsApp Image 2021-06-04 at 02 20 22

**Comment by iamdevvalecha** [comment]
@mariobehling @MeghalBisht please review.

**Comment by prsmahajan** [comment]
Hi, I would like to work on this issue, can you suggest me to get started?

**Comment by MeghalBisht** [comment]
Swap the position of text and arrow icon

**Comment by iamdevvalecha** [comment]
@MeghalBisht done.

**Comment by iamdevvalecha** [comment]
Screenshot from 2021-06-05 00-04-05

File: app/templates/components/tabbed-navigation.hbs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {{#if this.device.isMobile}}
   <div role="button" class="ui segment center aligned" {{action 'toggleMenu'}}>
     {{this.currentRoute}}
+      <i class="dropdown icon"></i>
```
Comment (MeghalBisht) [inline app/templates/components/tabbed-navigation.hbs:4]: indentation

Comment (iamdevvalecha) [reply app/templates/components/tabbed-navigation.hbs:4]: sorry for that. 1 min fixing!

File: app/templates/components/tabbed-navigation.hbs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {{#if this.device.isMobile}}
   <div role="button" class="ui segment center aligned" {{action 'toggleMenu'}}>
     {{this.currentRoute}}
+  <i class="dropdown icon"></i>
```
Comment (MeghalBisht) [inline app/templates/components/tabbed-navigation.hbs:4]: fix indentation

**Pull request merged**